docs/usage/build_fms_model.rst

```diff
@@ -76,28 +76,19 @@ The quickstart instructions can be used with the null model example located in t
 
 .. code-block::
 
-  # Create checkout script
-  fre make checkout-script -y null_model.yaml -p ncrc5.intel23 -t prod
-
   # Create and run checkout script
   fre make checkout-script -y null_model.yaml -p ncrc5.intel23 -t prod --execute
 
   # Create Makefile
   fre make makefile -y null_model.yaml -p ncrc5.intel23 -t prod
 
-  # Create the compile script
-  fre make compile-script -y null_model.yaml -p ncrc5.intel23 -t prod
-
   # Create and run the compile script
   fre make compile-script -y null_model.yaml -p ncrc5.intel23 -t prod --execute
 
 2. Bare-metal Build Multi-target:
 
 .. code-block::
 
-  # Create checkout script
-  fre make checkout-script -y null_model.yaml -p ncrc5.intel23 -t prod -t debug
-
   # Create and run checkout script
```

Comment:
more specific detail. "Create and run checkout script which will checkout the code as defined in the model yaml"